### PR TITLE
made changes for export Catalogue

### PR DESCRIPTION
**Pull request by arya-hemanshu**
### Description
exportCatalogue option was failing as it was built as per webComponent project and was expecting two arguments.

- made change to the gradle.properties.example file as properties related to the web project is not required anymore
- Changed gradle task accordingly
- Made changes to the CatalogueExportRunner w.r.t build.gradle
- Deleted unnecessary code from CatalogueExportRunner as that was required for web project.

### CODE REVIEW

**Comment by lorenaqendro** [comment]
@arya-hemanshu can you please add a short description of what was wrong and what approach does this PR propose?

**Comment by arya-hemanshu** [comment]
@lorenaqendro done :)

File: build.gradle [hunk 1]
```diff
@@ -113,22 +113,22 @@ task runCorrelation(dependsOn: ['classes'], type: JavaExec) {
...
+    doFirst {
         checkEnv()
         systemProperty("databaseURI", databaseURI)
         systemProperty("databaseUsername", databaseUsername)
         systemProperty("databasePassword", databasePassword)
         systemProperty("environment", "export")
 
-        def argumentsSet2 = project.hasProperty('outputFile')
+        def argumentsSet2 = project.hasProperty('output')
         if (!argumentsSet2) throw new GradleException('Required properties not specified. See the README.')
```
Comment (lorenaqendro) [inline build.gradle]: Feel free to change the name of the variable here to something like: `argumentSet`.

File: build.gradle [hunk 2]
```diff
@@ -113,22 +113,22 @@ task runCorrelation(dependsOn: ['classes'], type: JavaExec) {
...
-    doLast {
+    doFirst {
         checkEnv()
         systemProperty("databaseURI", databaseURI)
         systemProperty("databaseUsername", databaseUsername)
         systemProperty("databasePassword", databasePassword)
         systemProperty("environment", "export")
 
-        def argumentsSet2 = project.hasProperty('outputFile')
+        def argumentsSet2 = project.hasProperty('output')
```
Comment (lorenaqendro) [inline build.gradle]: Update README, since the property name has changed and add a one-liner there about the memory needed for this specific task.

**Pull request merged**